internal/controller/management/projects/projects.go

```diff
@@ -914,6 +915,52 @@ func (r *reconciler) ensureDefaultUserRoles(
 				},
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      promoterRoleName,
+				Namespace: project.Name,
+				Annotations: map[string]string{
+					rbacapi.AnnotationKeyManaged: rbacapi.AnnotationValueTrue,
+				},
+			},
+			Rules: []rbacv1.PolicyRule{
+				{ // For viewing events and serviceaccounts
+					APIGroups: []string{""},
+					Resources: []string{"events", "serviceaccounts"},
+					Verbs:     []string{"get", "list", "watch"},
+				},
+				{ // For viewing project-level access
+					APIGroups: []string{rbacv1.SchemeGroupVersion.Group},
+					Resources: []string{"rolebindings", "roles"},
+					Verbs:     []string{"get", "list", "watch"},
+				},
+				{ // View access to Kargo resources
+					APIGroups: []string{kargoapi.GroupVersion.Group},
+					Resources: []string{"freights", "stages", "warehouses", "projectconfigs"},
+					Verbs:     []string{"get", "list", "watch"},
+				},
+				{ // Promote permission on all stages
+					APIGroups: []string{kargoapi.GroupVersion.Group},
+					Resources: []string{"stages"},
+					Verbs:     []string{"promote"},
+				},
+				{ // Can create, view, and delete promotions
+					APIGroups: []string{kargoapi.GroupVersion.Group},
+					Resources: []string{"promotions"},
+					Verbs:     []string{"create", "delete", "get", "list", "watch"},
```

Comment:
Let's remove delete access. This isn't something users should normally be doing. If anyone does it, it should only be a project admin.